bin/node/src/service.rs

```diff
@@ -109,12 +109,7 @@ pub fn new_partial(
         })
         .transpose()?;
 
-    let executor = AlephExecutor::new(
-        config.wasm_method,
-        config.default_heap_pages,
-        config.max_runtime_instances,
-        config.runtime_cache_size,
-    );
+    let executor = sc_service::new_native_or_wasm_executor(config);
```

Comment:
I'm a bit confused, how does this create `AlephExecutor`? Or am I misreading that below we run a function that accepts an `AlephExecutor` as an argument?

Reply:
AlephExecutor is just a type alias:
```
pub type AlephExecutor = NativeElseWasmExecutor<ExecutorDispatch>;
```